layouts/shortcodes/deprecationwarning.html

```diff
@@ -5,7 +5,7 @@
       <h3>
 	 Kubernetes {{ .Page.Param "version" }}
 	 {{ T "deprecation_warning" }}
-	 <a href="{{ site.Params.currentUrl }}">{{ T "latest_version" }}</a>
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "latest_version" }}</a>
```

Comment:
I see other similar href are using `site.`, ping @tengqm for confirmation.

Reply:
LGTM